docs/protocol/modules/inflation.md

```diff
@@ -74,12 +71,12 @@ transaction with `unvested` tokens until they are unlocked represented as
 
 #### Exponential Inflation - **The Half Life**
 
-The inflation distribution for staking, usage incentives and community pool is
+The inflation distribution for staking and community pool is
```

Comment:
We recently added also the reduction factor = 3. It should be mentioned here

Reply:
Good point, I can add it.